static/app/views/organizationStats/usageStatsOrg.tsx

```diff
@@ -448,6 +462,7 @@ class UsageStatsOrganization<
         score={loading ? undefined : card.score}
         help={card.help}
         trend={card.trend}
+        isHoverable
```

Comment:
Would it make sense to set this via the card metadata and therefore only activate it for the tooltip that needs it?
```suggestion
        isTooltipHoverable={card.isTooltipHoverable}
```